### DISCUSSION TITLE
How can I get KlipperScreen to launch on startup within LXDE Desktop?

### DISCUSSION THREAD

**Discussion by Dweinbach** [KlipperScreen/KlipperScreen#1085] [Q&A]
Hi:
I'm running Klipperscreen as an app in the Raspberry pi desktop (with no known problems so far). I have this menu shortcut to launch it:

sh -c "$HOME/.KlipperScreen-env/bin/python $HOME/KlipperScreen/screen.py"

This works fin (added through main menu editor of desktop).

But I cannot figure out how to make it so desktop loads 1st, then launches klipperscreen automatically as a window. I've searched online and added that same line with an @ at the beginning to /etc/xdg/lxsession/LXDE-pi/autostart, but no joy.

I'm pretty weak in Linux OS, so any help would be appreciated!

**Comment by alfrix** [comment]
using the installer will result in autostarting and running as a service, running klipperscreen inside a desktop environment is not recommended and it should be used [only for development at the moment](https://klipperscreen.readthedocs.io/en/latest/Developers/)

**Comment by alfrix** [comment]
KS has matured enough that desktop mode is good enough, if anyone needs to autostart just create a .desktop file in ~/.config/autostart/
